ipatests/test_integration/test_sudo.py

```diff
@@ -750,3 +750,26 @@ def test_domain_resolution_order(self):
         finally:
             self.master.run_command(
                 ['ipa', 'config-mod', '--domain-resolution-order='])
+
+    def test_sudooption_not_failing(self):
```

Comment:
I'd replace "not_failing" with "display_members" since the bug was about not converting member DNs into more user-friendly strings.